modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/BaseProjectTemplatesTestCase.java

```diff
@@ -517,13 +571,175 @@ public default File buildWorkspace(
 
 		String name = "test-workspace";
 
-		File destinationDir = temporaryFolder.newFolder("workspace" + name);
+		File destinationDir = temporaryFolder.newFolder(
+			"gradleWorkspace" + name);
 
 		return buildTemplateWithGradle(
 			destinationDir, WorkspaceUtil.WORKSPACE, name, "--liferay-version",
 			liferayVersion);
 	}
 
+	public default File buildWorkspaceWithTPEnabled(
+			TemporaryFolder temporaryFolder, String buildType, String name,
+			String liferayVersion, MavenExecutor mavenExecutor)
+		throws Exception {
+
+		File workspaceDir;
+
+		if (buildType.equals("gradle")) {
+			workspaceDir = buildWorkspace(temporaryFolder, liferayVersion);
+
+			enableTargetPlatformInWorkspace(workspaceDir, liferayVersion);
+		}
+		else {
+			File destinationDir = temporaryFolder.newFolder("mavenWorkspace");
+			String groupId = "com.test";
+
+			workspaceDir = buildTemplateWithMaven(
+				destinationDir, destinationDir, "workspace", name, groupId,
+				mavenExecutor, "-DliferayVersion=" + liferayVersion,
+				"-Dpackage=com.test");
+		}
+
+		return workspaceDir;
+	}
+
+	public default void configureExecutePackageManagerTask(File projectDir)
+		throws Exception {
+
+		File buildGradleFile = new File(projectDir, "build.gradle");
+
+		StringBuilder sb = new StringBuilder();
+
+		String lineSeparator = System.lineSeparator();
+
+		sb.append(lineSeparator);
+
+		sb.append("buildscript {");
```

Comment:
We needed to add a buildscript { block to add the com.liferay.gradle.plugins to get com.liferay.gradle.plugins.node.tasks. ExecutePackageManagerTask, so it wasnt a bug. just something we need because of this import in test